### ISSUE TITLE
ENOENT error

### ISSUE THREAD

**Issue by gruzzilkin** [imagemin/cwebp-bin#7]
Hi! 

I use [gulp-webp](https://github.com/sindresorhus/gulp-webp). After last **`cwebp-bin`** update I see the `Error: spawn ENOENT` error message (on PNG files only).

I use Ubuntu 14.04, original `cwebp` utility works fine.

[CLI](https://github.com/imagemin/cwebp-bin#cli) output:
```
$ sudo npm install -g cwebp-bin
/usr/bin/cwebp -> /usr/lib/node_modules/cwebp-bin/cli.js

> cwebp-bin@2.0.3 postinstall /usr/lib/node_modules/cwebp-bin
> node install.js

  ✔ cwebp pre-build test passed successfully
npm WARN unmet dependency /usr/lib/node_modules/npm/node_modules/npm-package-arg requires semver@'4' but will load
npm WARN unmet dependency /usr/lib/node_modules/npm/node_modules/semver,
npm WARN unmet dependency which is version 2.3.0
cwebp-bin@2.0.3 /usr/lib/node_modules/cwebp-bin
├── logalot@2.1.0 (figures@1.3.5, squeak@1.2.0)
├── bin-wrapper@2.1.3 (os-filter-obj@1.0.0, lnfs@1.0.0, is-path-global@1.0.1, bin-check@1.0.0, globby@1.1.0, npm-installed@1.0.0, download-status@2.1.1, bin-version-check@2.1.0, download@3.3.0)
└── bin-build@2.1.1 (rimraf@2.2.8, tempfile@1.1.0, exec-series@1.0.1, url-regex@2.1.2, archive-type@2.0.0, decompress@2.1.1, download@3.3.0)
kolek@kolek:~/projects/7days/7days-redesign$ cwebp --help

events.js:72
        throw er; // Unhandled 'error' event
              ^
Error: spawn ENOENT
    at errnoException (child_process.js:1001:11)
    at Process.ChildProcess._handle.onexit (child_process.js:792:34)
```

**Comment by shinnn** [comment]
@gruzzilkin

I fixed this problem, unpublished v2.0.3 and released v2.0.4. Please update your local cwebp-bin to v2.0.4.

Thanks for reporting! :+1:

**Comment by gruzzilkin** [comment]
It worked, thanks!

**Comment by artemijeka** [comment]
`node node_modules/cwebp-bin/lib/install.js`
?

**Comment by ismatech** [comment]
hi guys|
I also get this issue but not on a local machine. I get it on CI/CD in GitLab during stage install_script.
And you know there is a build which is being deployed on docker container so the folder node_modules is always being removed. Maybe I should install some libs? I've seen one warning/error there: something like "didn't find any C compiler".
![cwebp-bin error](https://github.com/imagemin/cwebp-bin/assets/30582837/045ea126-590b-4ae5-8130-7a8df98c34b4)

**Comment by ismatech** [comment]
I found out why I face this problem. It's because in dockerfile we install from node:14-alpine version. 
If I just write from node:14, then with the command 
```docker
RUN apt update -y && \
    apt install -y  --no-install-recommends \
    ffmpeg && \
    rm -rf /var/lib/apt/lists/*
```
It is being installed correctly. Does anyone know how to install cwebp-bin with node14-alpine version?
UPD: devops guy just created separated dockerfile for node:14 in gitlab ci/cd, so we solved our problem. Hope, it will help you too.
